app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="recipes-show-container">
+  <h1>Add Ingredient</h1>
+  <%= form_with(model: @recipe_food, url: recipe_foods_path, local: true) do |form|%>
+    <span>
+      <%= form.collection_select :food_id, current_user.foods, :id, :name, prompt: 'Select a food' %>
+    </span>
+    <span>quantity: <%= form.number_field :quantity, min: 1 %></span>
+    <%= form.hidden_field :recipe_id, value: @recipe.id %>
+    <span><%= form.submit 'Add Ingredient', class: 'button-white-black' %></span>
+  <% end %>
+</div>
```

Comment:
- [Optional] Kindly keenly take a look at the screen cast below. It shows how a use could add ingredients with a success message while in reality the inputs fields where not fill in. No validation messages where displayed. I think this doesn't create a nice user's experience while using your app. Kindly validate all inputs.

[Screencast from 10-12-2023 08:08:14 AM.webm](https://github.com/danifromecuador/recipes/assets/63087491/a1f75ea5-a7bc-48c1-ac93-8e4d5cf3f46d)